pytorch/inference/docker/1.10/py3/Dockerfile.graviton.e3.cpu

```diff
@@ -120,7 +120,7 @@ RUN pip install pillow==8.3.2 "awscli<2"
 
 RUN pip uninstall -y model-archiver multi-model-server \
  && pip install captum \
- && pip install torchserve==$TS_VERSION \
+ && pip install torchserve-nightly==2022.3.23.post2 \
```

Comment:
This is great! Please also comment out the ARG for TS_VERSION on this dockerfile to prevent confusion at a later date.